polaris/ocean/tasks/isomip_plus/projection.py

```diff
@@ -0,0 +1,39 @@
+import pyproj
+
+
+def get_projection_string():
+    """
+    Get a stereographic projection string that should be used to define
+    latitudes and longitudes for the ISOMIP+ x-y coordinates
+
+    Returns
+    -------
+    proj_str : str
+        The projection string
+    """
+    # y_0 is half way through the domain so the longitude is approximately
+    # like -y
+    #
+    # x_0 is close to lat=-75 and lon=90 so latitude is approximately like x
+    proj_str = ('+proj=stere +lon_0=0 +lat_0=-90 +lat_ts=-75.0 +x_0=-2000e3 '
+                '+y_0=40.0e3 +ellps=WGS84')
```

Comment:
After thinking about the projection today, I decided I wasn't happy with the previous one (centered at lon=0, lat=-75) because it doesn't line up well with the x-y coordinates and it will be hard to make transects of.  This new coordinate leads to a `lonCell` that is symmetric about the middle of the domain and looks something like -y (varying mostly in the y direction), and a `latCell` that starts at about -75 degrees and looks something like x (varying mostly in the x direction).